libobs/media-io/video-io.h

```diff
@@ -119,20 +120,22 @@ static inline const char *get_video_format_name(enum video_format format)
 		return "I420";
 	case VIDEO_FORMAT_NV12:
 		return "NV12";
+	case VIDEO_FORMAT_I422:
+		return "I422";
 	case VIDEO_FORMAT_YVYU:
 		return "YVYU";
 	case VIDEO_FORMAT_YUY2:
 		return "YUY2";
 	case VIDEO_FORMAT_UYVY:
 		return "UYVY";
+	case VIDEO_FORMAT_I444:
+		return "I444";
 	case VIDEO_FORMAT_RGBA:
 		return "RGBA";
 	case VIDEO_FORMAT_BGRA:
 		return "BGRA";
 	case VIDEO_FORMAT_BGRX:
 		return "BGRX";
-	case VIDEO_FORMAT_I444:
-		return "I444";
```

Comment:
Was there a reason to move this case?

Reply:
match the order of the previous switch case, which follows a logical progression: 4:2:0, 4:2:2 (planar, packed), 4:4:4, uncompressed RGB formats

Reply:
I'll revert that